backend/src/main/java/com/bakdata/conquery/models/dictionary/DirectDictionary.java

```diff
@@ -29,6 +28,9 @@ public int size() {
 
 	@Override
 	public int getId(String value) {
+		if(dict == null)
```

Comment:
Wird nur in dieser Funktion gecheckt. Da das dict notwendig für die Klasse ist (was auch immer sie macht), würde ich das an derer Stelle testen.